reposerver/src/test/scala/com/advancedtelematic/tuf/reposerver/http/RepoResourceSpec.scala

```diff
@@ -460,14 +460,31 @@ trait RepoResourceSpec extends TufReposerverSpec with RepoSupport
     }
   }
 
+  test("POST on user_create creates a repository for a namespace with given KeyType") {
```

Comment:
why not use the new thing with multiple key types ?

Reply:
Because then it would be difficult to know whether the key type has been passed through HTTP or as the current one in the test.